tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -568,6 +569,11 @@ void CorProfiler::RewritingPInvokeMaps(const ModuleMetadata& module_metadata,
                         hr = metadata_emit->DefinePinvokeMap(methodDef, pdwMappingFlags,
                                                              shared::WSTRING(importName).c_str(),
                                                              profiler_ref);
+
+                        // Store this methodDef token in the internal tokens list
+                        auto intTokens = internal_rewrite_tokens.Get();
+                        intTokens->insert({ module_id, methodDef });
```

Comment:
Just for my clarity, we're deleting the original pinvoke map and creating a new one. So we don't need to add `[MethodImpl(MethodImplOptions.NoInlining)]` or anything anywhere, right?

Reply:
`[MethodImpl(MethodImplOptions.NoInlining)]` We don't add it mostly because there's nothing to inline, that method doesn't have body and PInvokes are resolved on runtime, not like LibraryImport. In fact I don't think is necessary to add them to the hashset, but I did it just in case some other weirdness happens